src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultConsentViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import Combine
+
+class ExposureSubmissionTestResultConsentViewController: DynamicTableViewController {
+	
+	
+	// MARK: - Init
+	
+	init(
+		supportedCountries: [Country],
+		exposureSubmissionService: ExposureSubmissionService
+	) {
+				
+		self.viewModel = ExposureSubmissionTestResultConsentViewModel(supportedCountries: supportedCountries, exposureSubmissionService: exposureSubmissionService)
+		super.init(nibName: nil, bundle: nil)
+	}
+	
```

Comment:
why use supportedCountries and exposureSubmissionService as parameters to only create a viewModel here?
You can create the viewmodel outside and pass into the init method

Reply:
Done.